extension/js/agent/utils/setHiddenProperty.js

```diff
@@ -1,10 +0,0 @@
-// @private
-var setHiddenProperty = function(object, property, value) {
-    if (!_.isObject(object)) return;
-    Object.defineProperty(object, hiddenPropertyPrefix+property, {
-        configurable: false,
-        enumerable: false,
-        value: value,
-        writable: true
-    });
-};
```

Comment:
Hmm... Maybe I'm a hoarder by nature, but I feel like it's nice to have this around incase we need it later. Also I know be debugger uses it a ton.